liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/StopServerMojo.java

```diff
@@ -28,7 +28,7 @@
 public class StopServerMojo extends StartDebugMojoSupport {
 
     /**
-     * Timeout to verify stop successfully
+     * Timeout to verify stop successfully, this parameter is ignored
```

Comment:
I don't think it will break existing scripts.  The plugin will ignore the configuration it doesn't understand.